mixins/labelled-mixin.js

```diff
@@ -0,0 +1,87 @@
+
+export const LabelMixin = superclass => class extends superclass {
+
+	static get properties() {
+		return {
+			_label: { type: String, reflect: true }
+		};
+	}
+
+};
+
+export const LabelledMixin = superclass => class extends superclass {
+
+	static get properties() {
+		return {
+			labelledBy: { type: String, reflect: true, attribute: 'labelled-by' },
+			label: { type: String }
+		};
+	}
```

Comment:
Wondering: if we had a setter/getter for `label` (that set a private `_label`), would that give us more info about whether they explicitly set the label vs. if we set it programatically based on `labelled-by`? That could let us have one always take precedence over the other. i.e. if they've set `label` explicitly maybe we just ignore `labelled-by`.

Reply:
Do you mean that we would have a separate private `_label` property?  The consumer needs to rerender itself when the label changes, and currently that rerender is triggered by changing the `label` property.  Having that property would keep things separate and allow us to make the consumer-specified `label` take precedence, but I'm not sure how just a setter for `label` would achieve that.  But yeah, currently if they were to specify both `labelledBy` and `label`, the `labelledBy` would take precedence, which I think would be ok since it would be strange to specify both.

Reply:
Yeah nevermind... thinking about it more I think it just adds more complexity than it's worth.